templates/jenkins/files/README.md

```diff
@@ -1,19 +1,37 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
-This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Jenkins projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
```

Comment:
This second part should be included in the description in the template.yaml, and both lines wrapped in p tags.  (Same for all of the templates.)

Reply:
Ok, Done, thank you